Example/HelloSDK/HelloVC.swift

```diff
@@ -97,6 +104,16 @@ final class HelloVC: UIViewController {
 			self?.mapPitchLabel.text = "Map pitch: \(pitch)"
 		}
 
+		self.map.floorPlanDidChange = { [weak self] currentLevelIndex in
+			if let currentLevelIndex = currentLevelIndex,
+			   self?.map.floorPlan?.currentLevelIndex == currentLevelIndex {
+				self?.mapFloorPlanLabel.text = "Floor plan: \(currentLevelIndex)"
```

Comment:
Тут лишний отступ. Не нужно стесняться, что строки сливаются. Если в общепринятом стиле есть проблемы, пусть все это видят. :)
```suggestion
			self?.mapFloorPlanLabel.text = "Floor plan: \(currentLevelIndex)"
```